src/Components/Products/index.js

```diff
@@ -1,10 +1,108 @@
 import React from 'react'
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
 import styles from './products.module.css';
+import Modal from '../Shared/Modal/index';
+
+const Products = (props) => {
+  const [products, setProducts] = useState([]);
+  const [showModal, setShowModal] = useState(false);
+  const [productId, setProductId] = useState(undefined);
+
+  useEffect(() => {
+    async function fetchProducts() {
+      try {
+        const response = await fetch(`${process.env.REACT_APP_API_URL}/products/`);
+        const json = await response.json();
+        setProducts(json.data);
+      } catch (error) {
+        alert('Could not GET Products.', error);
```

Comment:
handling this error with an alert isn't the best practice, remember alerts blocks the directional flow of work in js, blocking tasks, so you might want to just throw the error and use some cleaner way to show it, could be only a string in red

Reply:
This will be an modal later on. It is like this now so we could know if there is an error